estuary_updater/handlers/freshmaker.py

```diff
@@ -24,12 +24,26 @@ def can_handle(msg):
         :return: a bool based on if the handler can handle this kind of message
         :rtype: bool
         """
-        return msg['topic'] == '/topic/VirtualTopic.eng.freshmaker.event.state.changed'
+        return msg['topic'] == '/topic/VirtualTopic.eng.freshmaker.event.state.changed' or \
+            msg['topic'] == '/topic/VirtualTopic.eng.freshmaker.build.state.changed'
 
     def handle(self, msg):
         """
         Handle a Freshmaker message and update Neo4j if necessary.
 
+        :param dict msg: a message to be processed
+        """
+        topic = msg["topic"]
+
+        if topic == "/topic/VirtualTopic.eng.freshmaker.event.state.changed":
+            FreshmakerHandler.event_state_handler(self, msg)
+        elif topic == "/topic/VirtualTopic.eng.freshmaker.build.state.changed":
+            FreshmakerHandler.build_state_handler(self, msg)
```

Comment:
As a safe guard, could you add an `else` clause so that unhandled messages don't pass silently?